docs/conops.md

```diff
@@ -0,0 +1,146 @@
+
+Last but not least, certifications with different government agencies around the world can be difficult to navigate and require significant legal resources and even lobbying efforts. Diverting our limited resources and contributors' time on bureaucratic processes may drag down the progress of the entire organization, especially given that Arrow operates on an anonymity-first principle, which can draw extra scrutiny from aviation regulatory bodies.
+
+### Marketplace Platform
+An alternative business model would be a marketplace platform like that of Uber. Arrow would solicit bids from OEM players to supply their fleet to our network, and each OEM player would have the discretion to submit a bidding for their vehicles for particular trips.
+
+For example, if a customer wants to ship a 10-kg cargo from San Francisco to San Jose departing April 10th, Arrow would present a list of options. So, the customer may see Elroy, Joby, and Zipline vehicles available for this particular itinerary but at different pricing points:
+
+*(Illustrative purpose only)*
+* Joby Cargo: $47.2
+* Lilium Cargo: $50.1
+* Zipline Cargo: $51.3
+
+All else being equal, the customer would naturally pick the Joby option because it is the cheapest. This behavior will incentivize OEMs to innovate in order to compete with other players on price.
+
+The innovation can come from hardware (advanced manufacturing, superior procurement management, economic design, etc), or from algorithm. Some well-funded players may not care about recouping CapEx, so they would develop algorithms to analyze the supply and demand of the market, pricing of other services, etc., to programmatically price their trips.
+
+#### Pricing as a Service
+Arrow may offer a set of APIs for OEMs to develop in-house algorithms, but it could also be a product offering.
+
+Since Arrow has the full-access to everything on the network, it may makes sense to offer a white-labeled pricing algorithm solution to OEM players, which 1) provides an additional source of revenue to Arrow, and 2) help these OEM players to narrow their focus to the things that they are good at -- making vehicles.
+
+We believe this business model creates a win-win situation where it releases the production and legal burdens from Arrow, and it serves as a direct-to-consumer channel to OEMs, allowing them to monetize on their vehicles with a shorter time-to-market than if they have to sell them through contracts.
+
+
+#### Downside
+While the marketplace business model bridges many gaps on the market amount different participants, the platform (Arrow) itself may under-perform as operating a marketplace is difficult and the success is largely a function [network effects](https://online.hbs.edu/blog/post/what-are-network-effects). 
```

Comment:
function *of network effects

Reply:
[00d00e5](https://github.com/Arrow-air/svc-pricing/pull/3/commits/00d00e5af21d9fe993cefdae5033554b1ab6a211)